.cirrus.yml

```diff
@@ -115,7 +115,7 @@ build_each_commit_task:
         matrix:
             image_name: "fedora-29-libpod-9afa57a9"
 
-    timeout_in: 20m
+    timeout_in: 30m
```

Comment:
Sigh, ya, that's a lot of commits.  Ideally the timeout should scale based on the number of commits.  That could be accomplished "moving" the effective timeout into the Makefile (timeout command).  That's maybe is more work/troubleshooting/low-priority though.  The vast majority of PRs are small, and finish this in under 5 minutes, so bumping timeout here is an acceptable compromise.